### PR TITLE
Manage custom availability

### PR DESCRIPTION
**Pull request by benlovell**
Grants booking managers the ability to alter custom availability, based
on previouly built populated schedules.

New schedules wipe out previously saved slots.

### Schedules

schedules

### Calendar

calendar

### Availability

availability

### CODE REVIEW

**Comment by benlovell** [comment]
[As a booking manager I am able to manage bookable slots for a location's availability](https://trello.com/c/QKqUZ09U/159-as-a-booking-manager-i-am-able-to-manage-bookable-slots-for-a-locations-availability)

File: app/assets/javascripts/modules/availability-calendar.es6 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          $('body').append($rendered);
+        }
+      });
+    }
+
+    availabilityModal(date) {
+      var editUri = `${this.$editUri}?date=${date.format()}`;
+
+      // mark the calendar for reloading
+      $('.t-calendar-rendered').remove();
```
Comment (jongilbraith) [inline app/assets/javascripts/modules/availability-calendar.es6]: Should probably be using the `js-` prefix for this.

Comment (benlovell) [inline app/assets/javascripts/modules/availability-calendar.es6]: Yes! Good spot.

Comment (benlovell) [inline app/assets/javascripts/modules/availability-calendar.es6]: Oh, though its sole purpose is for tests.

**Pull request merged**